dask/dataframe/core.py

```diff
@@ -5054,8 +5054,17 @@ def eval(self, expr, inplace=None, **kwargs):
         meta = self._meta.eval(expr, inplace=inplace, **kwargs)
         return self.map_partitions(M.eval, expr, meta=meta, inplace=inplace, **kwargs)
 
+    if PANDAS_GT_150:
+        from pandas._libs.lib import no_default
```

Comment:
Nit: rather than importing from a private module, we can import from `pandas.api.extensions`. It's not a huge deal, and pandas doesn't provide very strong guarantees around this, it seems, but see some discussion in https://github.com/pandas-dev/pandas/issues/40397

Reply:
+1